client-sdk/go/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/oasisprotocol/oasis-sdk/client-sdk/go
 
-go 1.18
+go 1.19
```

Comment:
some of these down here only upgrade to 1.19. do they need a different version? although I see oasis-core does the same with go.mod files saying 1.19 while using 1.20

Reply:
Yeah we support both 1.19 and 1.20.